src/Carousel.js

```diff
@@ -122,214 +123,224 @@ let Carousel = React.createClass({
       index = ValidComponentChildren.count(this.props.children) - 1;
     }
 
-    this.handleSelect(index, e, 'prev');
-  },
-
-  pause() {
-    this.isPaused = true;
-    clearTimeout(this.timeout);
-  },
+    this.select(index, e, 'prev');
+  }
 
-  play() {
```

Comment:
You think so? They're not documented anywhere. They'll break as soon as we move this to use uncontrollable anyway. IMO if someone complains we'd be better off just adding a prop to control this.